modules/tinymce/src/plugins/table/main/ts/api/Settings.ts

```diff
@@ -21,6 +21,14 @@ const defaultStyles = {
   'border-collapse': 'collapse',
   'width': '100%'
 };
+const defaultCellBorderWidth = Arr.range(5, (i) => {
```

Comment:
```suggestion

const defaultCellBorderWidth = Arr.range(5, (i) => {
```
There's a missing new line here, so just adding that.